creator-node/src/utils/legacyUtils.js

```diff
@@ -355,6 +401,36 @@ export function currentNodeShouldHandleTranscode({
   return currentNodeShouldHandleTranscode
 }
 
+/**
+ * Verify that the file written matches the hash expected
+ * @param {Object} param
+ * @param {string} param.cid target cid
+ * @param {string} param.path the path at which the cid exists
+ * @param {Object} param.logger
+ * @returns boolean if the cid is proper or not
+ */
+export async function verifyCIDIsProper({ cid, path, logger }) {
```

Comment:
lets be explicit here - "verifyCIDIsProper" is vague - instead "verifyCIDMatchesExpected" or smth?